TI-reports/2025/2025-Q1-Vulnerability-Disclosure-WG.md

```diff
@@ -0,0 +1,30 @@
+# 2025 Q1 Vulnerability Disclosure WG - Madison Oliver
+
+## Overview
+TI is progressing as expected. Engagement has remained relatively stable with slight decreases over the last 6 months. We've had external presentations and discussions with the [package url](https://github.com/package-url/purl-spec) community and passed a vote to adopt Advise as a new project.
+
+## Vulnerability Dislosures Working Group
+
+### Purpose
+The OpenSSF Vulnerability Disclosures Working Group seeks to help improve the overall security of the open source software ecosystem by helping mature and advocate well-managed vulnerability reporting and communication.
+
+### Current Status
+- Typically have 5-6 regular attendees for biweekly full WG meeting, limited additional engagement in the monthly APAC meeting
+- [Graduated](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-become-graduated).
+- [To remain Graduated](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-remain-graduated):
+   - [x] Have received TAC approval of the README.md per Incubating requirements above
+   - [x] Have met at least 4 times over a period of at least 2 months since becoming Incubating
+   - [x] Have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
+   - [x] Request TAC approval. TAC will vote to approve or provide constructive guidance.
+   - [x] TAC sponsor monitoring and consultation become optional.
+
+### Up Next
+- WG Project Board: https://github.com/orgs/ossf/projects/29
+- [Project Idea - CVD Guide for OSS Consumers](https://github.com/ossf/wg-vulnerability-disclosures/issues/115) > effort is stagnating and needs to be revived
+- [VOTE - Adopt Advise as a Sandbox project for the OpenSSF](https://github.com/ossf/wg-vulnerability-disclosures/issues/152) > vote has passed, Madison needs to create a request for the TAC next
+- [Discussions regarding purl implementation in the CVE Program](https://github.com/ossf/wg-vulnerability-disclosures/issues/158) > discussed in the last four meetings with additional activity over the mailing list
+- [LF Member Summit](https://events.linuxfoundation.org/lf-member-summit/program/schedule/), March 18-20 > panel of OpenSSF members discussing open source vulnerabilities and how to bridge the gap between discovery and remediation
+- [VulnCon 2025](https://www.first.org/conference/vulncon2025/), April 7-10 > many WG members planning to attend with some likely speaking
+
+### Questions/Issues for the TAC
+- What other areas in the OpenSSF 2025 Roadmap does the TAC see opportunity for the Vulnerability Disclosures working group? Can we support more with CRA guidance? 👀
```

Comment:
I'm not directly involved, but I agree that there's an opportunity to collaborate with the Global Cyber Policy WG. Another idea that came to mind because it has come up in discussion a few times is how this WG may engage with the AI/ML Security WG, since guidance on finding and fixing AI/ML-specific vulnerabilities is on the roadmap as well.